react/features/device-selection/popup.js

```diff
@@ -3,10 +3,14 @@ import 'aui-experimental-css';
 
 import DeviceSelectionPopup from './DeviceSelectionPopup';
 
+declare var JitsiMeetJS: Object;
+
 let deviceSelectionPopup;
 
 window.init = function(i18next) {
-    deviceSelectionPopup = new DeviceSelectionPopup(i18next);
+    JitsiMeetJS.init({}).then(() => {
+        deviceSelectionPopup = new DeviceSelectionPopup(i18next);
+    });
```

Comment:
Is this the change that will wait for gum to complete to better guarantee gum has been called before device selection does its logic?